### ISSUE TITLE
Analytics: update referral labels?

### ISSUE THREAD

**Issue by jacqui** [news-catalyst/next-tinynewsdemo#1012]
I've been keeping tabs on the various GA data importers and note the referral sessions data could potentially be made a bit more user friendly by matching incoming domains:

* `n-s4vshwzfwlvnqx56yk3pzvazbnucuytmjbpbiga-1lu-script.googleusercontent.com`: I think this is the Google Docs sidebar
* `lm.facebook.com`: [facebook mobile link referrer](https://clix.co/faq/2014/09/05/facebook-referral-sites/), should just be facebook?
* `vercel.com`: will our founders understand what this is? maybe label vercel and sidebar referrers as internal or similar?

There are likely more examples - part of this work would be compiling a more comprehensive list.

**Comment by TylerFisher** [comment]
Yeah, I think this is a good idea, though it seems low priority to me.

**Comment by cbj0hns0n** [comment]
Repro: Go to Analytics > Sessions. See names for the sites, like "t.co." It would be helpful to rename these to make more sense to the end user.
